openshift-template-dev.yml

```diff
@@ -88,13 +88,44 @@ objects:
           containers :
             - name : ${APP_NAME}
               image : ${APP_NAME}:latest
+              ports:
+                - containerPort: 8080
+                  protocol: "TCP"
               resources:
                 requests:
                   memory: "64Mi"
                   cpu: "50m"
                 limits:
                   memory: "128Mi"
                   cpu: "100m"
+  - kind: "Service"
+    apiVersion: "v1"
+    metadata:
+      name: "release-bot"
+      labels:
+         application: "release-bot"
+      annotations:
+        description: "Exposes release-bot port for github webhook callbacks"
+    spec:
+      ports:
+        - name: "8080-tcp"
+          port: 8080
+          protocol: "TCP"
+          targetPort: 8080
+      selector:
+        deploymentconfig: ${APP_NAME}
+  - kind: "Route"
+    apiVersion: "v1"
+    metadata:
+      name: "release-bot"
+      labels:
+         application: "release-bot"
+      annotations:
+        description: "Github webhook route to reach release-bot"
+    spec:
+      to:
+        kind: "Service"
+        name: "release-bot"
```

Comment:
It's not necessary to enclose all the values in double quotes, i.e. you can have e.g.
```yaml
        kind: Service
        name: release-bot
```
instead of
```yaml
        kind: "Service"
        name: "release-bot"
```